recipe_batches/recipe_batches.py

```diff
@@ -1,10 +1,26 @@
 #!/usr/bin/python
 
 import math
+import operator
 
 def recipe_batches(recipe, ingredients):
-  pass 
-
+  # Need to get the max value from ingredients.items, this is a starting point
+  batch = max(ingredients.items(), key=operator.itemgetter(1))[1]
+  # Now I need to loop 
+  for recKey in recipe.items():
+    # if the key from recipes does not appear in ingredients, batch=0 because we can't make recipe
+    if recKey[0] not in ingredients.keys():
+      batch = 0
+    # loop through the ingredients dict
+    for ingKey in ingredients.items():
```

Comment:
this works fine! however, notice that it's 2 nested for loops (making this solution O(n**2). See if you can bring it down to O(n)